sympy/sets/sets.py

```diff
@@ -466,15 +466,16 @@ def powerset(self):
 
         A power set of an empty set:
 
-        >>> A = EmptySet()
+        >>> from sympy import FiniteSet, EmptySet
+        >>> A = EmptySet
         >>> A.powerset()
         {EmptySet()}
```

Comment:
This printing is needs to be fixed, right?

Reply:
That's fixed now. Some other doctests needed changing.